tools/_setup_generation/setup.py

```diff
@@ -101,19 +102,25 @@ def exit(self, setup: Setup):
     def setup(self, setup: Setup):
         pass
 
+
 from .step_viselements import VisElementsStep
 from .step_refman import RefManStep
 from .step_rest_refman import RestRefManStep
 from .step_gui_ext_refman import GuiExtRefManStep
 from .step_getting_started import GettingStartedStep
-
-def run_setup(root_dir: str, steps: List[SetupStep] = [
-    VisElementsStep(),
-    RefManStep(),
-    RestRefManStep(),
-    GuiExtRefManStep(),
-    GettingStartedStep()
-    ]):
+from .step_contributor import ContributorStep
```

Comment:
I would have called this step 'contributors' unless you plan for no more than one.
(and ContributorsStep and so on)

Reply:
Yep!